icu4c/source/test/hdrtst/testinternalheaders.sh

```diff
@@ -9,27 +9,38 @@
 CC=clang
 CXX=clang++
 
+ERROR_EXIT=0
```

Comment:
One thought is that we changed the script to use bash instead of sh then we could perhaps use `set -x -e`, so that the script would exit if any command fails with a non-zero exit status.
However, that would mean the script stops right away on the first error -- whereas I think the intent of the change here is to have *all* header checks run. (That way you don't end up with fixing one error, only to hit another, and so on, etc.)
Maybe a comment about this might be worth adding perhaps?

Reply:
I think we want to run all tests and get a deferred error. Otherwise you'd have to iterate over every header that's broken.

Reply:
Right, I was just wonder if it might be worth adding a comment about as such, but perhaps it's obvious from the script.